getting_started/first_app.md

```diff
@@ -142,6 +142,13 @@ Note that you may not see all the options listed below on your command line, bec
     - **Adobe Campaign Standard**: including dependencies and examples of accessing the [Adobe Campaign Standard (ACS) API](https://docs.adobe.com/content/help/en/campaign-standard/using/working-with-apis/get-started-apis.html)
     - **Adobe Experience Platform: Realtime Customer Profile**: including dependencies and examples of accessing the [Realtime Customer Profile API of Adobe Experience Platform](https://www.adobe.io/apis/experienceplatform/home/api-reference.html#!acpdr/swagger-specs/real-time-customer-profile.yaml)
     - **Generic**: a generic back-end action with hello world flow that can be reused and modified e.g. for simple serverless computing or 3rd party API integration
+    
+1. If you included `Web Assets` under Adobe I/O App features you will be given two choices. One to include React Spectrum based UI teamplate or a Vanilla HTML/JS one
```

Comment:
teamplate -> template

Could we link `[React Spectrum](https://react-spectrum.adobe.com/)`?

Could we mention that the Vanilla HTML/JS template is also based on `[Spectrum CSS](https://opensource.adobe.com/spectrum-css/)`?

Could we mention that both templates come with the boilerplate code needed to integrate the Project Firefly application within the `[Adobe Experience Cloud](https://github.com/AdobeDocs/project-firefly/blob/master/guides/exc_app/overview.md)`?